### ISSUE TITLE
Proposal for the FDC3 project to adopt the new FINOS Standards Governance model (including the Community Specification License)

### ISSUE THREAD

**Issue by agitana** [finos/FDC3#412]
This GitHub issue serves the following purposes:

1. Socialize this proposal
2. Collect input and questions from the FDC3 Community about the proposal.
3. Outline the steps to implement this proposal.

### Proposal
The FDC3 project should consider adopting the recently ratified [FINOS Open Standards project Governance](https://github.com/finos/community/tree/master/governance#open-standard-projects).

The project will bring this proposal up for discussion and voting at an upcoming FDC3 Standard Working Group meeting (**_date TBD_**). If you would like to vote on the decision to adopt this new governance model, please [enroll as an FDC3 Voting Participant](https://github.com/finos/fdc3#enrollment).

In the meantime, we are socializing this proposal for the FDC3 community to get familiar with it, and we will be using this GitHub issue to collect questions and concerns from the community, and to share further relevant updates.

### Ask to the Community
Please review the [FINOS Open Standards project Governance](https://github.com/finos/community/tree/master/governance#open-standard-projects) and the [Community Specification License](https://github.com/finos/standards-project-blueprint), and **comment on this GitHub issue with questions or concerns about** FDC3 adopting the [FINOS Open Standards project Governance](https://github.com/finos/community/tree/master/governance#open-standard-projects). 

### Background
The FDC3 project is currently using the ["old" IP policy based standard governance](https://github.com/finos/community/tree/db11c11b80651b8bc888e7cb1d56d20afdf2a346/governance/Standards-Projects). 

In April 2021, the FINOS Governing Board approved the usage of the [Community Specification](https://github.com/finos/standards-project-blueprint) as the default charter for open standard projects within the Foundation. This governance model brings together the best of two worlds, enabling an open standard process to be carried out in familiar open source tooling (like GitHub) and collaboration primitives (e.g. pull requests). 

We expect new FINOS open standard projects to adopt the new [FINOS Open Standards project Governance](https://github.com/finos/community/tree/master/governance#open-standard-projects) and [Community Specification License](https://github.com/finos/standards-project-blueprint), and it's advisable that existing FINOS standard projects, like FDC3, barring major objections consider moving to this model in the near future. 

### High level steps to implement this proposal
- _TBD_

**Comment by openfin-gavin** [comment]
I've raised a separate issue that is related to this one which suggests strengthening policies around IP - the issue is at https://github.com/finos/FDC3/issues/513

**Comment by kriswest** [comment]
Reopening as project is not complete

**Comment by Julia-Ritter** [comment]
Hi all, @mindthegab will be following up in the issue with an update on the CLA adoption

**Comment by mindthegab** [comment]
Hey @kriswest and @finos/fdc3-maintainers / @finos/fdc3-participants, sorry it took so long to come back here.

As per my update on https://github.com/finos/standards-project-blueprint/pull/4#issuecomment-1079395612, https://github.com/finos/standards-project-blueprint/pull/8#issuecomment-1079395898 and https://github.com/finos/standards-project-blueprint/pull/10#issuecomment-1079395986, currently three changes to the CSL are being evaluated by the Governing Board and the vote will complete on April 8th (or worse case at the GB meeting of April 20th). I'll provide an update back to the Community with the outcome.

Either the FDC3 project should wait to adopt the CSL, because, aside from these potential changes, it's also important to make a clean switch right after cutting the 2.0 release (before any further work goes into a future version). This will allow a clear set of licensing commitments for the 2.0 (IP policy) and 2.x/3.x (based on CSL, assuming it's adopted by this project).

That said, given FDC3 is the only project left on the IP policy, assuming the 2.0 release is cut in Q2, I would strongly recommend to move to the CSL ahead of the July GB meeting, so we can proceed removing the IPR provisions from the IP policy during that Board cycle.

Let us know if you have any further question.

**Comment by mindthegab** [comment]
Hey @kriswest @finos/fdc3-maintainers and @finos/fdc3-participants:

to close the loop here, [one](https://github.com/finos/standards-project-blueprint/pull/10) of the three CSL amendments has been approved by the Governing Board on April 20th, and the other two were rejected ([here](https://github.com/finos/standards-project-blueprint/pull/4) and [here](https://github.com/finos/standards-project-blueprint/pull/8)).

At this point there there are no more pending changes at FINOS level (unless you feel strongly we MUST approve downstream [this enhancement](https://github.com/CommunitySpecification/Community_Specification/pull/10) @kriswest and so, once 2.0 is done, this issue can and should be prioritized for merging.

Happy to work with your legal team for the correct process and disclosures.

**Comment by kriswest** [comment]
Many thanks for following up @mindthegab (both here and other threads about when we should do this). 

Regarding the minor enhancement to the wording of the CSL, I think we should approve and merge it downstream. I would assume its not particularly controversial (simple acknowledging that a standard can come in a form other than a single long document) and it makes the form of the FDC3 standard acceptable according to its own governance documentation. 

If you/FINOS don't have objections to it I can raise a new PR to the [standards-project-blueprint](https://github.com/finos/standards-project-blueprint).

**Comment by mindthegab** [comment]
> If you/FINOS don't have objections to it I can raise a new PR to the [standards-project-blueprint](https://github.com/finos/standards-project-blueprint).

@kriswest - absolutely, makes sense to me. We can try to do that at the next GB meeting.

**Comment by kriswest** [comment]
I believe this is now complete with the switch to the CSL @robmoffat 
Closing as complete
